Godeps/Godeps.json

```diff
@@ -7,6 +7,11 @@
 	],
 	"Deps": [
 		{
+			"ImportPath": "github.com/Microsoft/go-winio",
+			"Comment": "v0.3.8",
```

Comment:
Change to `v0.4.1`.
https://github.com/containerd/containerd/blob/8ed1e24ae925b5c6d8195858ee89dddb0507d65f/vendor.conf#L35